tests/sender/send_data.py

```diff
@@ -1,9 +1,14 @@
 import unittest
 import socket
+import time
 from ssl import CERT_NONE
 from devo.sender import Sender, SenderConfigTCP, SenderConfigSSL
 from devo.common import get_log
-from .load_certs import *
+
+try:
+    from .load_certs import *
+except ImportError:
+    from load_certs import *
```

Comment:
question(blocking): can you please explain why are you doing this best-effort approach? In which case could this fail? In which case could we expect to have a `load_certs` package in the `PYTHONPATH`?

Reply:
If all the tests are run there is no problem. If only one module is tested, some of the modules fail. I guess this will need future refinement.